client/src/utils/api.js

```diff
@@ -0,0 +1,5 @@
+export const getData = async (dataTag) => {
+  const response = await fetch(`/api/sven/${dataTag}`);
```

Comment:
You wouldn't have a custom API Endpoint for every user
```suggestion
  const response = await fetch(`/api/users/${user}/${dataTag}`);
```

Reply:
👍